core/geometry/src/test/curve/Arc3d.test.ts

```diff
@@ -650,6 +650,29 @@ describe("Arc3d", () => {
     expect(ck.getNumErrors()).equals(0);
   });
 
```

Comment:
I moved this nice function outside `EllipseSampler` test so I can use it in my test.